lib/superset/request.rb

```diff
@@ -33,7 +33,12 @@ def superset_host
     end
 
     def query_params
-      [filters, pagination].join
+      [filters, pagination, order_by].join
+    end
+
+    def order_by
+      # by default, we will order by changed_on as per the GUI
+      ",order_column:changed_on,order_direction:desc"
```

Comment:
this default value is removed in a later commit that sets default to nil
unfortunately Superset does not have a consistent naming convention for changed_on date
so we can not use this across all child classes